examples/ephemeral-spark/versions.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_version = "> 0.13"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.36, !=4.0.0, !=4.1.0, !=4.2.0, !=4.3.0, !=4.4.0, !=4.5.0, !=4.6.0, !=4.7.0, !=4.8.0"
```

Comment:
can we cleanup these versions files, it should only need to specify the provider versions in the module itself. Also not sure what is trying to be limited here in terms of versions, maybe `>4.8.0`?